.github/workflows/llvm.yml

```
@@ -36,6 +36,10 @@ jobs:
           version: "${{ matrix.llvm_version }}"
           cached: ${{ steps.cache-llvm.outputs.cache-hit }}
 
+      - name: Install libtinfo5
+        run: apt-get install -y libtinfo5
+        if: "${{ matrix.llvm_version == '16.0.0' }}"
```

No need for interpolation:

```suggestion
        if: matrix.llvm_version == "16.0.0"
```